libs/components/router/testing/src/href/href-harness.ts

```diff
@@ -0,0 +1,42 @@
+import { HarnessPredicate } from '@angular/cdk/testing';
+import { SkyComponentHarness } from '@skyux/core/testing';
+
+import { SkyHrefHarnessFilters } from './href-harness-filters';
+
+export class SkyHrefHarness extends SkyComponentHarness {
+  static hostSelector = '[skyHref]';
+
+  /**
+   * Gets a `HarnessPredicate` that can be used to search for a
+   * `SkyHrefHarness` that meets certain criteria.
+   */
+  public static with(
+    filters: SkyHrefHarnessFilters
+  ): HarnessPredicate<SkyHrefHarness> {
+    return this.getDataSkyIdPredicate(filters);
+  }
+
+  async getHref(): Promise<string | null> {
+    return this.forceStabilize().then(async () => {
```

Comment:
👇 Are these `forceStabilize` calls needed? I removed them and the tests still pass.

Reply:
I think I was chasing a different issue and hadn't removed them. They're not needed. Removed.